cfgov/cfgov/urls.py

```diff
@@ -74,243 +81,229 @@ def empty_200_response(request, *args, **kwargs):
+        re_path(r'^server-error/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/server-error/index.html'),
             name='server_error')],
         namespace='govdelivery')),
 
-    url(r'^regulation-comment/new/$', regsgov_comment, name='reg_comment'),
+    re_path(r'^regulation-comment/new/$', regsgov_comment, name='reg_comment'),
 
-    url(r'^regulation-comment/', include([
-        url(r'^success/$',
-            TemplateView.as_view(
-                template_name='regulation-comment/success/index.html'),
+    re_path(r'^regulation-comment/', include([
+        re_path(r'^success/$', TemplateView.as_view(
+            template_name='regulation-comment/success/index.html'),
             # 'core.views.comment_success',
             name='success'),
-        url(r'^error/$',
-            TemplateView.as_view(
-                template_name='regulation-comment/error/index.html'),
+        re_path(r'^error/$', TemplateView.as_view(
+            template_name='regulation-comment/error/index.html'),
             name='user_error'),
-        url(r'^server-error/$',
-            TemplateView.as_view(
-                template_name='regulation-comment/server-error/index.html'),
+        re_path(r'^server-error/$', TemplateView.as_view(
+            template_name='regulation-comment/server-error/index.html'),
             name='server_error')],
         namespace='reg_comment')),
 
-    url(r'^feed/$',
-        RedirectView.as_view(url='/about-us/blog/feed/', permanent=True)),
-    url(r'^feed/blog/$',
-        RedirectView.as_view(url='/about-us/blog/feed/', permanent=True)),
-    url(r'^feed/newsroom/$',
-        RedirectView.as_view(url='/about-us/newsroom/feed/', permanent=True)),
-    url(r'^newsroom-feed/$',
-        RedirectView.as_view(url='/about-us/newsroom/feed/', permanent=True)),
+    re_path(r'^feed/$', RedirectView.as_view(
+        url='/about-us/blog/feed/', permanent=True)),
+    re_path(r'^feed/blog/$', RedirectView.as_view(
+        url='/about-us/blog/feed/', permanent=True)),
+    re_path(r'^feed/newsroom/$', RedirectView.as_view(
+        url='/about-us/newsroom/feed/', permanent=True)),
+    re_path(r'^newsroom-feed/$', RedirectView.as_view(
+        url='/about-us/newsroom/feed/', permanent=True)),
 
-    url(r'^careers/(?P<path>.*)$', RedirectView.as_view(
+    re_path(r'^careers/(?P<path>.*)$', RedirectView.as_view(
         url='/about-us/careers/%(path)s', permanent=True)),
 
-    url(r'^transcripts/', include([
-        url(r'^how-to-apply-for-a-federal-job-with-the-cfpb/$',
+    re_path(r'^transcripts/', include([
+        re_path(r'^how-to-apply-for-a-federal-job-with-the-cfpb/$',
             TemplateView.as_view(
                 template_name='transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/index.html'),  # noqa: E501
                 name='how-to-apply-for-a-federal-job-with-the-cfpb'), ],
         namespace='transcripts')),
```

Comment:
Same as above, this will need a [two-tuple with an app namespace](https://docs.djangoproject.com/en/2.2/ref/urls/#include):

 ```python
   re_path(r'^transcripts/', include(
        (
            [
                re_path(r'^how-to-apply-for-a-federal-job-with-the-cfpb/$',
                        TemplateView.as_view(
                            template_name='transcripts/how-to-apply-for-a-federal-job-with-the-cfpb/index.html'),  # noqa: E501
                        name='how-to-apply-for-a-federal-job-with-the-cfpb'),
            ],
            'transcripts'
        ),
        namespace='transcripts')
    ),
```

Reply:
Updated `transcripts` regular expression to a two-tuple with an application namespace